### PR TITLE
docs(eth payments): add factory and top up examples PE-5992

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,32 +183,59 @@ const turbo = TurboFactory.unauthenticated();
 
 Creates an instance of a client that accesses Turbo's authenticated and unauthenticated services. Requires either a signer, or private key to be provided.
 
+##### Construct Turbo with a JWK
```
Comment (arielmelendez) [inline README.md:186]: * `with an Arweave JWK`

File: src/types.ts [hunk 2]
```diff
@@ -185,12 +185,24 @@ export type TurboPostBalanceResponse =
...
 
-export const isJWK = (wallet: TurboWallet): wallet is JWKInterface =>
-  (wallet as JWKInterface).kty !== undefined;
+type HexadecimalString = string;
+export type EthPrivateKey = HexadecimalString;
+
+export function isEthPrivateKey(wallet: TurboWallet): wallet is EthPrivateKey {
+  if (typeof wallet !== 'string') return false;
+
+  return wallet.startsWith('0x');
```
Comment (arielmelendez) [inline src/types.ts:199]: Is there any more wholistic validation that can be done here?

File: src/types.ts [hunk 3]
```diff
@@ -185,12 +185,24 @@ export type TurboPostBalanceResponse =
...
+}
+
+export type TurboWallet = ArweaveJWK | SolSecretKey | EthPrivateKey;
+
+export const isJWK = (wallet: TurboWallet): wallet is ArweaveJWK =>
+  (wallet as ArweaveJWK).kty !== undefined;
```
Comment (arielmelendez) [inline src/types.ts:205]: These checks are pretty light. Are the just intended to be a quick sniff test?

**Review by arielmelendez — APPROVED** [review]

File: src/types.ts (hunk 2 above)
Comment (fedellen) [reply src/types.ts:199]: so far these are mostly just to differentiate between SOL, ETH, and AR wallet types in our if check

**Pull request merged**